development/image-syncer/README.md

```diff
@@ -0,0 +1,63 @@
+# Image Syncer
+
+## Overview
+
+Image Syncer is used to **safely** copy container images from one registry to another. 
+The main use-case is to preserve images from third party registries in our own registry that we can relay on.
+
+Syncing process steps:
+- pull image from source
+- check if image exists in target
+- if it does not exist - re-tag image and push to target registry
+- if it exists compare IDs - if they are different synchronisation is interrupted, 
+it means that something is wrong - source image was changed (that should not happen)
```

Comment:
1. Since these are steps that should be performed in a strict order, this should be an ordered (numbered) list. See the [guidelines](https://github.com/kyma-project/community/blob/master/guidelines/content-guidelines/04-formatting.md#ordered-and-unordered-lists).
2. `synchronisation`-> `synchronization` - please use the American English spelling (see the [guidelines](https://github.com/kyma-project/community/blob/master/guidelines/content-guidelines/08-style-and-terminology.md#terminology)).
```suggestion
1. Pull image from source.
2. Check if image exists in target.
3a. If image does not exist: re-tag image and push to target registry.
3b. If image exists: compare IDs. If they are different, synchronization is interrupted. It means that something is wrong and the source image was changed (this should not happen).
```